File: 1653/1653.go
func minimumDeletionsToMakeStringBa(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumDeletionsToMakeStringBa(root.Left)
	r := minimumDeletionsToMakeStringBa(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 1653/1653.cs
public class Solution {
    public int MinimumDeletionsToMakeStringBa(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumDeletionsToMakeStringBa(root.left), MinimumDeletionsToMakeStringBa(root.right));
    }
}